src/agent/docker-rest-agent/requirements.txt

```diff
@@ -1,3 +1,22 @@
-Flask
-flask-restful
-requests
\ No newline at end of file
+aniso8601==8.0.0
+certifi==2020.6.20
+chardet==3.0.4
+click==7.1.2
+docker==4.3.1
+Flask==1.1.2
+Flask-RESTful==0.3.8
+idna==2.10
+itsdangerous==1.1.0
+Jinja2==2.11.2
+MarkupSafe==1.1.1
+pytz==2020.1
+requests==2.24.0
+six==1.15.0
+urllib3==1.25.11
+websocket-client==0.57.0
+Werkzeug==1.0.1
+gevent==20.9.0
+greenlet==0.4.17
+gunicorn==20.0.4
+zope.event==4.5.0
+zope.interface==5.1.2
```

Comment:
ok, I am wondering if all those packages are necessary？

Reply:
In the production environment, gunicorn is generally used to deploy the flask service

Reply:
Yep, I understand the usage of `gunicorn` but what I mean is that since you added multiple dependencies (3->22), are all those dependencies necessary? Well, all can keep for now, until some one is not used anymore. Thanks.

Reply:
Not all of them are necessary. For example, jinja2 will also be installed when you want to install flask. the requirements.txt is produced automatically；If you feel that you don't have to, you can leave them out, and it won't have any effect.